bitnami/prometheus-operator/templates/prometheus/additionalPrometheusRules.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.additionalPrometheusRules}}
+apiVersion: v1
+kind: List
+items:
+{{- range .Values.prometheus.additionalPrometheusRules }}
+  - apiVersion: monitoring.coreos.com/v1
+    kind: PrometheusRule
+    metadata:
+      name: {{ template "prometheus-operator.name" $ }}-{{ .name }}
+      namespace: {{ $.Release.Namespace }}
+      labels: {{ include "prometheus-operator.prometheus.labels" $ | nindent 8 }}
+    spec:
+      groups:
+{{ toYaml .groups| indent 8 }}
```

Comment:
I can see you are using templates in the `values.yaml` examples, because of that please consider using the following function here. It will render a template from the values and convert it to yaml: https://github.com/bitnami/charts/blob/d88de83e04f7dd3a1d1c99df54208439d9cc7913/bitnami/prometheus-operator/templates/_helpers.tpl#L274

Reply:
Hi, my PR changes are working, I tested. But I have difficulties with this improvement.

Reply:
ok. No problem, maybe it is harder to do it here because of the `range`.